docs/how-to-guides/curating-the-editor-experience/disable-editor-functionality.md

```diff
@@ -6,6 +6,37 @@ This page is dedicated to the many ways you can disable specific functionality i
 
 There might be times when you don’t want access to a block at all to be available for users. To control what’s available in the inserter, you can take two approaches: [an allow list](/docs/reference-guides/filters/block-filters.md#using-an-allow-list) that disables all blocks except those on the list or a [deny list that unregisters specific blocks](/docs/reference-guides/filters/block-filters.md#using-a-deny-list).
 
+## Curate heading levels
```

Comment:
>  It might be but I feel there's a lot of similar APIs that we need to surface in a consistent way (together).

💯 these attributes are not documented well. I'll add this to my to-do list.